modules/update-best-practices.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-best-practices_{context}"]
+= Best practices for cluster updates
+
+{product-title} is designed to provide a robust update experience that allows clusters to update with minimal disruptions to workloads.
+Updates will not begin unless the cluster is determined to be in an upgradeable state at the time of the update request.
+
+This design ensures that some key conditions are met before initiating an update, but there are a number of actions you can take to increase your chances of a successful cluster update.
```

Comment:
```suggestion
This design enforces some key conditions before initiating an update, but there are a number of actions you can take to ensure a successful cluster update.
```
Reworded passive voice and added stronger tone. "increase your chances" sounds like the user is gambling

Reply:
The first part of your suggestion sounds good, but I'm actively avoiding using terms like "ensuring" to avoid making promises to the user about the success of an update. The spirit of this doc is to acknowledge factors that can affect the success of an update and inform users how they can mitigate risk factors, it's not to assuage users that their updates will be universally successful.

IMO while we wouldn't want to explicitly state that updates are a gamble, that's sort of the nature of major tasks that introduce change and instability to the cluster, such as an update, and we don't want to misrepresent that nature in a doc meant to give users best practices and advice.